javascript/javascript1/week2/flight-booking.js

```diff
@@ -1,9 +1,14 @@
-function getfullname(firstname, lastname, useformal){
-  fullName = firstname + '' + lastname;
-  if (useformal) {
-    fullName = 'MR '+ fullName;
-    return fullName;
-  }
+let fullName1;
+let fullName2;
+let formalFullName = true;
```

Comment:
These variable "formalFullName" is also taken as parameters in the function, so it's not necessary to use it again. 

If you would like to assign a variable you can do it in functions local scope, it's not necessary to initialize on global scope